flexinput/src/main/java/com/lytefast/flexinput/adapters/PhotoCursorAdapter.kt

```diff
@@ -39,6 +56,10 @@ class PhotoCursorAdapter(private val contentResolver: ContentResolver,
 
   override fun onAttachedToRecyclerView(recyclerView: RecyclerView) {
     super.onAttachedToRecyclerView(recyclerView)
+
+    placeholderDrawable = ContextCompat.getDrawable(recyclerView.context, R.drawable.ic_image_24dp)
```

Comment:
I feel like maybe we can just use the empty drawable? or the placeholder drawable. either should be fine but we shouldn't need both.

Reply:
I added the empty drawable because the placeholder looked significantly worse, would prefer that.

Reply:
Okay, I think we can just pick one though right?